### DISCUSSION TITLE
Just switch to Terminal window with `wt` command

### DISCUSSION THREAD

**Discussion by tesar-tech** [microsoft/terminal#14295] [Q&A] [answered]
I am trying to bind shortcut key (using AHK) to open WT.. It is simple, when wt is not launch yet (just `wt` command). But when the terminal is out of focus it always opens new tab with the bound shortcut (and wt command).

There are ways how to target terminal window with AHK, but I am specifically asking about `wt` command - is it possible to not open new tab? Just focus the window..

**Comment by zadjii-msft** [comment, marked as answer]
What you're asking for is tracked in #13276
